code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -757,78 +757,48 @@ func (ssc *StorageSmartContract) GetValidatorHandler(ctx context.Context,
 
 }
 
-type BlobberOpenChallengesResponse struct {
-	BlobberID                string            `json:"blobber_id"`
-	ChallengeIDs             []string          `json:"challenge_ids"`
-	LatestCompletedChallenge *StorageChallenge `json:"lastest_completed_challenge"` // TODO: fix typo with Blobber and gosdk
+type StorageChallengeInfo struct {
```

Comment:
The postfix `*Info` actually does not give much info just like `Data`  and `util`, etc. This is a struct for API response, so I suggest to use `StorageChallengeResponse` or something similar if better.